### ISSUE TITLE
failure on stopping plesk services

### ISSUE THREAD

**Issue by locus313** [plesk/centos2alma#177]
**Describe the bug**
Getting a failure when the "rule plesk services" is running. Looks to be because postfix is not found. We do not use postfix on our plesk servers, we are instead using MSMTP to relay through our external email server. 
```
2024-02-16 18:02:44,440 - INFO - Do: rule plesk services!
2024-02-16 18:02:44,440 - INFO - Running: ['/usr/bin/systemctl', 'stop', 'crond.service', 'fail2ban.service', 'httpd.service', 'mariadb.service', 'mysqld.service', 'plesk-ext-monitoring-hcd.service', 'plesk-task-manager.service', 'plesk-web-socket.service', 'psa.service', 'sw-collectd.service', 'sw-cp-server.service', 'sw-engine.service', 'postfix.service']. Output:
2024-02-16 18:02:44,598 - INFO - Failed to stop postfix.service: Unit postfix.service not loaded.
2024-02-16 18:02:49,411 - ERROR - Command '['/usr/bin/systemctl', 'stop', 'crond.service', 'fail2ban.service', 'httpd.service', 'mariadb.service', 'mysqld.service', 'plesk-ext-monitoring-hcd.service', 'plesk-task-manager.service', 'plesk-web-socket.service', 'psa.service', 'sw-collectd.service', 'sw-cp-server.service', 'sw-engine.service', 'postfix.service']' failed with return code 5
2024-02-16 18:02:49,412 - ERROR - Failed: rule plesk services!. The reason: Command '['/usr/bin/systemctl', 'stop', 'crond.service', 'fail2ban.service', 'httpd.service', 'mariadb.service', 'mysqld.service', 'plesk-ext-monitoring-hcd.service', 'plesk-task-manager.service', 'plesk-web-socket.service', 'psa.service', 'sw-collectd.service', 'sw-cp-server.service', 'sw-engine.service', 'postfix.service']' returned non-zero exit status 5.
2024-02-16 18:02:53,070 - ERROR - centos2alma process has been failed. Error: Failed: rule plesk services!. The reason: Command '['/usr/bin/systemctl', 'stop', 'crond.service', 'fail2ban.service', 'httpd.service', 'mariadb.service', 'mysqld.service', 'plesk-ext-monitoring-hcd.service', 'plesk-task-manager.service', 'plesk-web-socket.service', 'psa.service', 'sw-collectd.service', 'sw-cp-server.service', 'sw-engine.service', 'postfix.service']' returned non-zero exit status 5.
```

**Feedback archive**
Here is the feedback archive
[centos2alma_feedback.zip](https://github.com/plesk/centos2alma/files/14317446/centos2alma_feedback.zip)

**Comment by SandakovMM** [comment]
The problem should be fixed with #180 merged.
Please find attached centso2alma with the fix:  [centos2alma.zip](https://github.com/plesk/centos2alma/files/14373337/centos2alma.zip)

**Comment by locus313** [comment]
I tested the version with the fix and no longer get the error
